content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: OpenGitOps 1.0 is finally here and why you should care
+date: "2021-11-03T09:00:01.284Z"
+description: "OpenGitOps 1.0 brings a mature standard to the DevOps industry."
+author: Dan Garfield and Scott Rigby
+thumbnail: images/GitOps-1.0.jpg
+---
+
+GitOps is a term coined by Alexis Richardson a few years ago and has grown immensely in popularity.
+In a nutshell, GitOps is a set of principles for operating and managing software systems, derived from modern software operations, but also rooted in pre-existing and widely adopted best practices.
+However they differ in specific ways from traditional CI/CD pipeline approach that has dominated the industry for decades.
+Last year, Amazon, Azure, Codefresh, Github, Redhat, and Weaveworks teamed up to launch the GitOps Working Group under the CNCF.
+The goal was to formalize all the ideas that have developed around GitOps into a single cohesive set of principles that would be easily digestible and repeatable by any organization.
+
+We’re very proud to announce that over 60 companies, 96 interested parties, and 34 co-authors have brought the best of DevOps into the [OpenGitOps 1.0 Principles and Glossary](https://github.com/open-gitops/documents/releases/tag/v1.0.0).
+Each principle discusses the desired state of a system and how it should be operated.
+
+The desired state of a GitOps managed system must be:
+
+1. **Declarative**
+
+    A [system](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#software-system) managed by GitOps must have its desired state expressed [declaratively](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#declarative-description).
+
+2. **Versioned and Immutable**
+
+    Desired state is [stored](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#state-store) in a way that enforces immutability, versioning and retains a complete version history.
+
+3. **Pulled Automatically**
+
+    Software agents automatically pull the desired state declarations from the source.
+
+4. **Continuously Reconciled**
+
+    Software agents [continuously](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#continuous) observe actual system state and [attempt to apply](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#reconciliation) the desired state.
+
+## GitOps is more than just storing infrastructure-as-code
+
+As [git history shows](https://github.com/open-gitops/documents/commits/v1.0.0), the wording of each principle and linked glossary item was very carefully chosen.
+While many of these principles should be familiar to DevOps professionals, many are surprised to learn that they still have work to do to implement GitOps fully.
```

Comment:
```suggestion
While many of these principles should be familiar to professionals, many are surprised to learn that they still have work to do to implement GitOps fully.
```
@scottrigby Would you accept this suggestion?  This suggestion from https://github.com/open-gitops/website/pull/45#discussion_r742245403 got lost in your rebase.